src/vos/evt_priv.h

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2017-2019 Intel Corporation.
+ * (C) Copyright 2017-2020 Intel Corporation.
```

Comment:
This is an artifact of changing this file in my patch...I'd rather leave it than remove it since I'm going to change this file in subsequent patch anyway